src/language-js/options.js

```diff
@@ -26,13 +26,14 @@ module.exports = {
       },
     ],
   },
+  bracketSameLine: commonOptions.bracketSameLine,
   bracketSpacing: commonOptions.bracketSpacing,
   jsxBracketSameLine: {
     since: "0.17.0",
     category: CATEGORY_JAVASCRIPT,
     type: "boolean",
-    default: false,
     description: "Put > on the last line instead of at a new line.",
+    deprecated: "2.4.0",
```

Comment:
We had test for deprecated options before, can we add one for this?  Check https://github.com/prettier/prettier/pull/7536/files, the old `tests_integration/__tests__/arg-parsing.js` file is moved to `tests/integration/__tests__/arg-parsing.js`